code/modules/mob/living/carbon/human/human.dm

```diff
@@ -112,6 +115,9 @@
 
 	if(new_species_name)
 		s_tone = random_skin_tone(new_species_name)
+	skin_r = rand(0,255)
```

Comment:
Then can you put these under an if which checks for the flag? It's not clear what this code is for otherwise.